app/controllers/staff_info_controller.rb

```diff
@@ -0,0 +1,80 @@
+class StaffInfoController < ApplicationController
+  before_action :authorize_prison_request
+
+  STAFF_INFO_ENDPOINT = ENV.fetch('STAFF_INFO_ENDPOINT', nil)
+
+  before_action :not_found, unless: :endpoint_set?
+  before_action :send_pdf, if: :pdf?
+  before_action :fetch_nav, only: :show
+  before_action :proxy_content, only: :show
+
+  def index
+    @content = proxy_content
+    render :index, status: req.status
+  end
+
+  def show
+    @content = markdown? ? convert_markdown : proxy_content
+    case req.status
+    when 200 then render :show
+    when 404 then render 'errors/404', status: 404
+    when 500 then render 'errors/500', status: 500
+    when 503 then render 'errors/503', status: 503
+    end
+  end
+
+private
+
+  def endpoint_set?
+    STAFF_INFO_ENDPOINT.present?
+  end
+
+  def not_found
+    render 'errors/404', status: 404
+  end
+
+  def fetch_nav
+    endpoint = "#{STAFF_INFO_ENDPOINT}nav.md"
+    @nav = markdown.render(Excon.get(endpoint).body.scrub!)
+  end
+
+  def markdown?
+    req.headers['Content-Type'] == 'text/markdown'
+  end
+
+  def send_pdf
+    send_data(proxy_content, filename: page_params, type: 'application/pdf')
+  end
+
+  def pdf?
+    req.headers['Content-Type'] == 'application/pdf'
+  end
+
+  def convert_markdown
+    unless proxy_content.encoding == Encoding::UTF_8
+      proxy_content.force_encoding('UTF-8')
+    end
+    proxy_content.scrub! unless proxy_content.valid_encoding?
+    markdown.render(proxy_content)
+  end
+
+  def proxy_content
+    @proxy_content ||= req.body
+  end
+
+  def remote_endpoint(endpoint = page_params)
+    "#{STAFF_INFO_ENDPOINT}#{endpoint}"
+  end
+
+  def page_params
+    params.permit(:page, :format).values.compact.join('.')
+  end
+
+  def req
+    @req ||= Excon.get(remote_endpoint)
```

Comment:
This is a niggle, but it does mean we're not using a persistent TCP connection for S3 connections. It doesn't matter given the traffic volume, but it would have been better to create a long-lived connection like all the other code using excon.